utils/ssh.py

```diff
@@ -135,10 +148,14 @@ def get_transport(self, *args, **kwargs):
             self.connect()
         return super(SSHClient, self).get_transport(*args, **kwargs)
 
-    def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False):
+    def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, ensure_host=False):
         if isinstance(command, dict):
             command = version.pick(command)
         logger.info("Running command `%s`", command)
+        if self._container and not ensure_host:
```

Comment:
It somehow makes sense to me to use the same variable for check and for the value later ... dunno. Of course I can change it.